### PR TITLE
Python program that surfs  3 site at a time

### PR DESCRIPTION
**Pull request by Kunal614**
add input in the compiling time  like --  python3 project1.py (man)

### CODE REVIEW

File: project1.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+from bs4 import BeautifulSoup
+import requests
+#from urllib.request import urlopen
+import webbrowser
+import sys
+from fake_useragent import UserAgent
```
Comment (cclauss) [inline project1.py]: https://travis-ci.com/TheAlgorithms/Python/builds/132468677#L334

Comment (Kunal614) [reply project1.py]: Please explain I don't understand what to do and how to do.

Comment (Kunal614) [reply project1.py]: I did't get error

Comment (cclauss) [reply project1.py]: Comment out the line that it is complaining about and then see if the script still runs.  flake8 is saying that the import is not needed.

**Comment by Kunal614** [comment]
it is needed otherwise program will work for 2 - 3 times then stop working.

File: project1.py (hunk 1 above)
Comment (Kunal614) [reply project1.py]: > Comment out the line that it is complaining about and then see if the script still runs. flake8 is saying that the import is not needed.

commented

**Pull request merged**